### PR TITLE
Adds native menu and credential details view

### PR DESCRIPTION
**Pull request by amanji**
* Adds react-native-menu for more natural looking context menu
* Some code refactoring and rearrangement
* Adds credential details screen
* Updates credhelper with new functions

### CODE REVIEW

File: src/utils/credhelper.ts [hunk 1]
```diff
@@ -21,52 +21,109 @@ export enum ImmunizationStatus {
...
       name,
     )}, ${this.givenNameForCredential(name)}`;
   }
 
-  public static nameForCredential(item: SHCRecord): PersonName | string {
-    const results = item.vc.credentialSubject.fhirBundle.entry.filter(
-      (e: any) => e.resource.resourceType === FhirBundleResourceType.Patient,
-    );
+  public static nameForCredential(item: SHCRecord): PersonName | undefined {
+    const results = this.patientForRecord(item);
```
Comment (jleach) [inline src/utils/credhelper.ts:63]: Does `this` work in the context of a `static` fn? I though we needed to use `CredentialHelper` like `CredentialHelper..patientForRecord(item);`. I think I missed a few also.

**Review by jleach — APPROVED** [review]

File: src/utils/credhelper.ts (hunk 1 above)
Comment (amanji) [reply src/utils/credhelper.ts:63]: I think you can https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static#calling_static_members_from_another_static_method

Comment (amanji) [reply src/utils/credhelper.ts:63]: But there's this caveat https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static#calling_static_members_from_a_class_constructor_and_other_methods

Comment (amanji) [reply src/utils/credhelper.ts:63]: You can only use the `this` keyword if calling from another static method

**Pull request merged**